encrypted-media/idlharness.https.html

```diff
@@ -22,16 +22,19 @@ <h1 class="instructions">Description</h1>
     <script>
       'use strict';
 
-      promise_test(async () => {
-        const idls = await fetch('/interfaces/encrypted-media.idl').then(r => r.text());
-        const html = await fetch('/interfaces/html.idl').then(r => r.text());
-        const dom = await fetch('/interfaces/dom.idl').then(r => r.text());
-        var idl_array = new IdlArray();
-        idl_array.add_idls(idls);
-        idl_array.add_dependency_idls(html);
-        idl_array.add_dependency_idls(dom);
-        idl_array.test();
-      }, 'Test encrypted-media IDL');
+      idl_test(
+        ['encrypted-media'],
+        ['html', 'dom'],
+        idl_array => {
+          self.audio = document.createElement('audio');
+          self.video = document.createElement('video');
+          idl_array.add_objects({
+            HTMLAudioElement: ['audio'],
+            HTMLVideoElement: ['video'],
+            Navigator: ['navigator'],
```

Comment:
I added the `Navigator` object here, and it resulted in two more subtests as expected. The audio and video elements don't do anything though, presumably because they inherit from `HTMLMediaElement` and idlharness isn't looking for instances of inherited types. @lukebjerring can you confirm?

Reply:
Alright, I should remove the useless instances then.

Reply:
Sent https://github.com/web-platform-tests/wpt/pull/18694